src/neo/SmartContract/Native/Tokens/Nep5Token.cs

```diff
@@ -118,11 +118,16 @@ public virtual BigInteger BalanceOf(StoreView snapshot, UInt160 account)
         }
 
         [ContractMethod(0_08000000, CallFlags.AllowModifyStates)]
-        protected virtual bool Transfer(ApplicationEngine engine, UInt160 from, UInt160 to, BigInteger amount)
+        protected bool Transfer(ApplicationEngine engine, UInt160 from, UInt160 to, BigInteger amount)
         {
-            if (amount.Sign < 0) throw new ArgumentOutOfRangeException(nameof(amount));
             if (!from.Equals(engine.CallingScriptHash) && !engine.CheckWitnessInternal(from))
                 return false;
+            return TransferInternal(engine, from, to, amount);
+        }
+
+        internal virtual bool TransferInternal(ApplicationEngine engine, UInt160 from, UInt160 to, BigInteger amount)
```

Comment:
This change it's related to oracles?

Reply:
It's not related now. I used it to send gas from the contract to oracle nodes. But now, I mint them directly. So I will revert this change.